spec/acceptance/manage/bzr_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'vcsrepo::manage::bzr', if: ['Debian', 'Archlinux'].include?(os[:family]) do
+  # bzr not present on RHEL or SLES
```

Comment:
I'd go with the "negative" approach, to avoid typos. It becomes self-documenting and if you do make a typo (like `Deban` instead of `Debian`) you don't accidentally disable testing on a platform. With the negative approach the tests will fail and you're pointed to your mistake, thus reducing your changes to mess up.
```suggestion
describe 'vcsrepo::manage::bzr', unless: ['RedHat', 'SLES'].include?(os[:family]) do
```